src/JointVelocityCalculator.cpp

```diff
@@ -0,0 +1,143 @@
+#include "JointVelocityCalculator.h"
+
+
+JointVelocityCalculator::JointVelocityCalculator(
+    ros::NodeHandle *nodeHandle,
+    tf2_ros::Buffer *tfBuffer,
+    vector<string> *jointNames,
+    void (*onVelocityCalculated)(string, double)
+
+) : nodeHandle(nodeHandle),
+    tfBuffer(tfBuffer),
+    jointNames(jointNames),
+    onVelocityCalculated(onVelocityCalculated)
+{
+    // Nothing to do here
```

Comment:
what does it means?

Reply:
The class members are initialized via an initializer list instead of using traditional assignments like `this->jointNames = jointNames`.
You can find more detailed information [here](https://www.learncpp.com/cpp-tutorial/8-5a-constructor-member-initializer-lists/) :wink: